packages/orchestrator-ui-components/src/pages/metadata/WfoTasksPage.tsx

```diff
@@ -0,0 +1,230 @@
+import React, { useEffect, useState } from 'react';
+
+import { useTranslations } from 'next-intl';
+
+import { EuiBadgeGroup } from '@elastic/eui';
+import type { Pagination } from '@elastic/eui/src/components';
+
+import { GET_TASKS_GRAPHQL_QUERY } from '@/graphqlQueries/tasksQuery';
+import {
+    useDataDisplayParams,
+    useQueryWithGraphql,
+    useQueryWithGraphqlLazy,
+    useShowToastMessage,
+    useStoredTableConfig,
+} from '@/hooks';
+import type { GraphqlQueryVariables, TaskDefinition } from '@/types';
+import { BadgeType, SortOrder } from '@/types';
+import {
+    getQueryVariablesForExport,
+    onlyUnique,
+    parseDateToLocaleDateTimeString,
+    parseIsoString,
+} from '@/utils';
+import {
+    csvDownloadHandler,
+    getCsvFileNameWithDate,
+} from '@/utils/csvDownload';
+
+import {
+    DEFAULT_PAGE_SIZE,
+    DEFAULT_PAGE_SIZES,
+    METADATA_TASKS_TABLE_LOCAL_STORAGE_KEY,
+    WfoProductBlockBadge,
+} from '../../components';
+import { WfoTableWithFilter } from '../../components';
```

Comment:
Use @/components

Reply:
Check, a copy paste from workflows page, I will fix that page as well.